tests/unit/test_genotype_phenotype.py

```diff
@@ -7,16 +7,19 @@
 from __future__ import unicode_literals
 
 import unittest
+import rdflib
 
 import ga4gh.datamodel.genotype_phenotype as g2p
 import ga4gh.datamodel.datasets as datasets
 
+# from ga4gh.protocol import FeaturePhenotypeAssociation
+
 
 class TestPhenotypeAssociationSet(unittest.TestCase):
     def setUp(self):
         ds = datasets.AbstractDataset("test")
         self.phenotypeAssocationSet = g2p.PhenotypeAssociationSet(
-            ds, "test", None)
+            ds, "test", '/server/tests/data/datasets/dataset1/phenotypes/cgd')
```

Comment:
This is the correct path?  It works.... At the same time I would have guessed the path would start with "tests/data/..."
I'm thinking that this works because `/server` is true for our docker image.  I did a quick experiment.  It works with the path set to "tests/..."  can you try it on your side and if it works, change it?